test/e2e/suite/approval/approval.go

```diff
@@ -82,7 +85,7 @@ var _ = framework.CertManagerDescribe("Approval CertificateRequests", func() {
 					Resources: []string{"certificaterequests"},
 				},
 				{
-					Verbs:     []string{"update"},
+					Verbs:     []string{"patch"},
```

Comment:
I wonder why we create a separate `Role` for each test? They will all have the same verbs and resources AFAICS.
Maybe add a TODO note to clean this up if you agree.

Reply:
As I understand each of these tests creates it's own service account and RBAC and tests that the service account can/cannot approve/deny the CR with that RBAC, so the creation of the Role is part of the test. Perhaps I didn't quite understand what you meant?